异常的处理try...catch

try{
    尝试执行的代码
}catch(Exception e){
    处理异常的代码
}finally{
    最终的
}